db/migrate/20160602153918_increase_decimal_precision.rb

```diff
@@ -0,0 +1,11 @@
+class IncreaseDecimalPrecision < ActiveRecord::Migration
+  def up
+    change_column :price_policies, :overage_rate, :decimal, precision: 12, scale: 4
+    change_column :price_policies, :reservation_rate, :decimal, precision: 12, scale: 4
```

Comment:
`usage_rate` is already at 12/4, and we had specs with rates like 0.002777777777777783333333333, which translate to zero if the field is 10/2.